README.md

```diff
@@ -1,94 +1,92 @@
-LicenseAdapter
-===
+# License Adapter
```

Comment:
I prefere LicenseAdapter

Reply:
SGTM, I thought it was a typo or something. 😄